testkit/src/mill/testkit/BspServerTestUtil.scala

```diff
@@ -0,0 +1,186 @@
+package mill.testkit
```

Comment:
`testkit` is meant for things that we publish for the rest of the world to use. Given that this stuff is for testing our own BSP implementation, it should probably just live in that module's test suite.

Reply:
Sure. I was thinking this might be useful in other tests (multi-module ones maybe), but we can always move that elsewhere later if needed.